### PR TITLE
Request app-operator v3.2.1 for KVM

### PR DESCRIPTION
**Pull request by rossf7**

### CODE REVIEW

File: kvm/requests.yaml [hunk 1]
```diff
@@ -5,15 +5,18 @@ releases:
...
           version: ">= 1.4.0"
           issue: https://github.com/giantswarm/giantswarm/issues/13771
     - name: "> 13.1.0"
       requests:
+       - name: app-operator
+          version: ">= 3.2.1 < 4.0.0"
+          issue: https://github.com/giantswarm/giantswarm/issues/15920
         - name: cluster-operator
-          version: ">= 0.23.22"
+          version: ">= 0.23.22 < 0.24.0"
```
Comment (rossf7) [inline kvm/requests.yaml:19]: cluster-operator 0.24.0 needs cert-operator 1.0.1.

File: kvm/requests.yaml [hunk 2]
```diff
@@ -5,15 +5,18 @@ releases:
...
+          version: ">= 0.24.2 < 0.25.0"
           issue: https://github.com/giantswarm/giantswarm/issues/14205
         - name: coredns
           version: ">= 1.4.0"
           issue: https://github.com/giantswarm/giantswarm/issues/13771
     - name: "> 13.1.0"
       requests:
+       - name: app-operator
+          version: ">= 3.2.1 < 4.0.0"
+          issue: https://github.com/giantswarm/giantswarm/issues/15920
```
Comment (rossf7) [inline kvm/requests.yaml:17]: This includes a security fix for the status webhook ingress.

File: kvm/requests.yaml [hunk 3]
```diff
@@ -5,15 +5,18 @@ releases:
           version: ">= 1.0.1"
           issue: https://github.com/giantswarm/giantswarm/issues/14205
         - name: cluster-operator
-          version: ">= 0.24.2"
+          version: ">= 0.24.2 < 0.25.0"
```
Comment (rossf7) [inline kvm/requests.yaml:8]: cluster-operator 0.25.0 has changes for the per cluster app-operator (v4.0.0 onwards).

But there is a problem. kvm-operator won't delete the cluster namespace because there is the app-operator pod. 🤦‍♂️ 

Will need to do another patch release of the legacy cluster-operator to delete the app CRs. On other providers they are deleted with the namespace.

**Review by rossf7 — COMMENTED** [review]
Tested on gorgoth.

**Review by tomahawk28 — APPROVED** [review]
